tests/mocks.cpp

```diff
@@ -75,67 +75,58 @@ void setup_mock_probe_matcher(MockProbeMatcher &matcher)
 void setup_mock_bpftrace(MockBPFtrace &bpftrace)
 {
   // Fill in some default tracepoint struct definitions
-  bpftrace.structs_["struct _tracepoint_sched_sched_one"] = Struct{
-    .size = 8,
-    .fields = { { "common_field",
-                  Field{
-                      .type = CreateUInt64(),
-                      .offset = 8,
-                      .is_bitfield = false,
-                      .bitfield = {},
-                  } } },
-  };
-  bpftrace.structs_["struct _tracepoint_sched_sched_two"] = Struct{
-    .size = 8,
-    .fields = { { "common_field",
-                  Field{
-                      .type = CreateUInt64(),
-                      .offset = 16, // different offset than
-                                    // sched_one.common_field
-                      .is_bitfield = false,
-                      .bitfield = {},
-                  } } },
-  };
-  bpftrace.structs_["struct _tracepoint_sched_extra_sched_extra"] = Struct{
-    .size = 8,
-    .fields = { { "common_field",
-                  Field{
-                      .type = CreateUInt64(),
-                      .offset = 24, // different offset than
-                                    // sched_(one|two).common_field
-                      .is_bitfield = false,
-                      .bitfield = {},
-                  } } },
-  };
-  bpftrace.structs_["struct _tracepoint_tcp_some_tcp_tp"] = Struct{
-    .size = 16,
-    .fields = { { "saddr_v6",
-                  Field{
-                      .type = CreateArray(16, CreateUInt(8)),
-                      .offset = 8,
-                      .is_bitfield = false,
-                      .bitfield = {},
-                  } } },
-  };
+  bpftrace.structs.Add("struct _tracepoint_sched_sched_one", 8);
```

Comment:
Kinda weird now that I think about it to have `struct` keyword in key. It's it's a huge diff to change that then don't bother.

Reply:
We're using the original type name collected by `ClangParser` so we'd have to have a special handling for tracepoint defs, which I think doesn't make much sense.